### ISSUE TITLE
Why did training stop?

### ISSUE THREAD

**Issue by Lavistas** [AlexeyAB/darknet#3762]
Hi! I am training from scratch and the whole process stopped after the "Used FMA & AVX2", as shown 
![image](https://user-images.githubusercontent.com/51375624/62972974-e1045f80-be1d-11e9-91da-ea282322e65d.jpeg)
in the picture. Does this mean that i train only on CPU? I have NVIDIA GEFORCE 930MX. Can someone help me?

**Comment by keko950** [comment]
Can you show your cfg?

**Comment by Lavistas** [comment]
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=8
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 6000
policy=steps
steps=4800,5400
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=24
activation=linear

[yolo]
mask = 3,4,5
#anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
anchors =25,137, 311, 67,  66,329, 142,159, 366,141, 193,287, 363,246, 299,350, 388,366
classes=3
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=24
activation=linear

[yolo]
mask = 0,1,2
#anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
anchors = 25,137, 311, 67,  66,329, 142,159, 366,141, 193,287, 363,246, 299,350, 388,366
classes=3
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

**Comment by Lavistas** [comment]
Thank u for your answer. I realized that i was training my data on CPU due to wrong compile of darknet. I followed the right instruction and now i train on GPU!
